lib/config.js

```diff
@@ -0,0 +1,5 @@
+const config = {
+    GITHUB_API_URL: 'https://api.github.com'
+};
+
+module.exports = config;
```

Comment:
Nice! But it's better to create folder `/lib/config` and put the api config inside that folder as `api.js`